cpp/Platform.Interfaces/CSet.h

```diff
@@ -52,9 +61,9 @@ namespace Platform::Interfaces
 
     }
 
-    template<typename Self, typename... Item>
-    concept CSet = CEnumerable<Self> && Internal::CSetHelpFunction<Self, Item...>();
+    template<typename TSelf, typename... TItem>
```

Comment:
All parameter packs should be named in plural form of the noun.
`TItem` → `TItems`